tips/templates/tips/board.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Tips</title>
+	<style type="text/css">
+	  .author, .eom { color: grey }
+	</style>
+  </head>
+  <body>
+    <h1>Tips</h1>
+<li><a href="add_tip/">Add new tip</a></li>
+{% for tip in tips %}
```

Comment:
I think that it will be better if it will be possible to see the tips and not as a link. 
I saw this page line boars with notes in my mind